src/main/java/org/minions/devfund/dennis/MovieDatabase.java

```diff
@@ -0,0 +1,105 @@
+package org.minions.devfund.dennis;
+
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * This is the methods for MovieDatabase class.
+ */
+public class MovieDatabase {
+    private ArrayList<Movie> movieList;
+    private ArrayList<Actor> actorList;
+
+    /**
+     * Constructor for MovieDatabase.
+     */
+    public MovieDatabase() {
+        movieList = new ArrayList<>();
+        actorList = new ArrayList<>();
+    }
+
+    /**
+     * Get the actor list.
+     * @return actor list
+     */
+    public ArrayList<Actor> getActorList() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)